e-commerce_fake-store/src/pages/home-page/index.js

```diff
@@ -12,7 +12,7 @@ const Home = () => {
   const api = "https://fakestoreapi.com/";
 
   const getAllProducts = useCallback(async () => {
-    const response = await fetch(`${api}` + "products");
+    const response = await fetch(`${api}products`);
```

Comment:
@shubhamshukla91 Will this API URL work properly? `${api}products`.

Reply:
yeah since '/' has been provided in the link already.
https://fakestoreapi.com/